src/main/java/uk/gov/companieshouse/itemhandler/exception/RetryableErrorException.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.itemhandler.exception;
+
+public class RetryableErrorException extends RuntimeException {
+    /**
+     * Thrown to indicate some error in processing that can be be recovered from if tried again. For example, a network
+     * connectivity error that may go away during subsequent retries.
+     * @param message error message
+     */
```

Comment:
Could this be a class Javadoc?

Reply:
Yes. That is where it needs to be. Thanks. Updated.